README.md

```diff
@@ -348,6 +349,12 @@ If you need to specify the path to header files for a system package installed b
 -Xcc -I$BUILD_DIR/.apt/usr/include/<path to header files>
 ```
 
+If you need to specify environment variables to pass to the build, such as for a [Kitura-NIO dependency](https://github.com/IBM-Swift/Kitura-NIO#using-kitura-nio), create a file with the name `.build_env` in the root project directory. Edit the file and add one environment variable per line. For example:
```

Comment:
I'd word this as:
> If you need to specify environment variables that apply during the build, such as to enable `Kitura-NIO` (link), create a file ... etc

Also, we noticed #119 - if you are changing the name of the file, it needs to be consistent here.  (Although both Helen and I have issues with the proposed name - see my comment in the other PR)